.travis/fedora.sh

```diff
@@ -10,46 +10,59 @@ docker exec travis-ci bash -c "dnf update -y"
 docker exec travis-ci bash -c "dnf install -y git rpm-build gcc-c++ cmake python3-devel python3-numpy"
 docker exec travis-ci bash -c "pip3 install cython scikit-build"
 
-# install OpenCL environment
-docker exec travis-ci bash -c "dnf install -y opencl-headers hwloc-devel libtool-ltdl-devel ocl-icd-devel ocl-icd clang llvm-devel clang-devel zlib-devel blas-devel boost-devel patch --setopt=install_weak_deps=False"
-docker exec travis-ci bash -c "git clone https://github.com/clMathLibraries/clBLAS.git"
-docker exec travis-ci bash -c "cd ./clBLAS/src && cmake . -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_TEST=OFF -DBUILD_KTEST=OFF"
-docker exec travis-ci bash -c "cd ./clBLAS/src && make && make install"
-# pocl 0.13 does not contain mem_fence() function that is used by primitiv.
-# We build the latest pocl instead of using distribution's package.
-# See: https://github.com/pocl/pocl/issues/294
-docker exec travis-ci bash -c "git clone https://github.com/pocl/pocl.git"
-docker exec travis-ci bash -c "cd ./pocl && cmake . -DCMAKE_INSTALL_PREFIX=/usr"
-docker exec travis-ci bash -c "cd ./pocl && make && make install"
+# OpenCL test is disabled in the following reasons:
+#
+# 1. Irreproducible memory error is occured in Python+OpenCL+Fedora(+Travis?) combination.
+# 2. The test is run on the virtual machine. This situation is very rare.
```

Comment:
Remove this line, and add some documents what the developer should do to solve this problem.